src/webcomponents/helperfunctions/jsonToCol.ts

```diff
@@ -0,0 +1,34 @@
+import { JsonToCol, LanguagesInput } from "../../@types/interfaces";
+
+const mapLangToSheetColumns = (languages: LanguagesInput) => {
+  const columnTitle = Object.keys(languages).map((title) => ({
+    type: "text",
+    title,
+    width: 150,
+  }));
+  return columnTitle;
+};
+
+const mapLangToSheetData = (languages: LanguagesInput) => {
+  const messageIDs = Object.keys(languages).map((titles) => languages[titles]);
+  const firstLangMessageIDs = Object.keys(messageIDs[0]);
+  const rows = messageIDs.map((messageID: object) =>
```

Comment:
do not use `: object` nowhere in this project (I am repeating over and over again)
type of this variable is inferred